pyproject.toml

```diff
@@ -45,6 +45,7 @@ pandas = ">=1.0"
 pre-commit = {version = ">=2.7.1", optional = true}
 prospector = {version = "*", optional = true}
 pydata-sphinx-theme = {version = ">=0.4.0", optional = true}
+pyro-ppl = ">=1.1.0"
```

Comment:
We'd probably like to keep this optional

Reply:
I thought about this, but they have really minimal requirements, and it might confusing for some people why they need to install pyro. In other words, the end users don't care if it's pytorch or pyro. 

https://github.com/pyro-ppl/pyro/blob/51d230ff62aab731f93bab66993a5e3532b749d0/setup.py#L85

Reply:
sounds good to me